request.go

```diff
@@ -244,6 +244,19 @@ func unmarshalNode(data *Node, model reflect.Value, included *map[string]*Node)
 				continue
 			}
 
+			if v.Kind() == reflect.String {
+				stringValue := v.Interface().(string)
+
+				// The field may or may not be a pointer to a string; handle both cases
+				if fieldValue.Kind() == reflect.Ptr {
+					fieldValue.Set(reflect.ValueOf(&stringValue))
```

Comment:
Hmm, I was thinking that it would assign it the addr of a zero string and then it would be non-nil.  Is this wrong?  Forgive my ignorance if any is obvious here :)